xbmc/cores/VideoPlayer/DVDSubtitles/webvtt/WebVTTHandler.cpp

```diff
@@ -206,26 +192,42 @@ bool CWebVTTHandler::Initialize()
   AddDefaultCssClasses();
 
   // Compile regex patterns
+  if(!m_signatureRegex.RegComp(signaturePattern))
+  {
+    return false;
+  }
```

Comment:
all these added brackets from here and below are not needed please revert them

here we dont have the rule of having mandatory brackets for a single code line (in this case the return)